### PR TITLE
Functions - Sum Of Multiples

### PR DESCRIPTION
**Pull request by Abduliitr**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [x] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [x] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [x] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [x] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

**Review by LVSsandeepVarma — APPROVED** [review]

File: solutions/functions/sum-of-multiples.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+//Write  your function here
+
+function sum_of_multiples(list, multiple){
```
Comment (sailesh-ramayanam) [inline solutions/functions/sum-of-multiples.js:15]: @Abduliitr 
1. Please update the function name to `camelCase`. It is updated in the template.
2. Rename `multiple` to `divisor`.

File: solutions/functions/sum-of-multiples.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            result += list[i];
+        }
+    }
+
+    return result;
+}
+
+
+// DO NOT CHANGE ANYTHING BELOW THIS LINE
+var range =parseInt(readLine())
```
Comment (sailesh-ramayanam) [inline solutions/functions/sum-of-multiples.js:30]: @Abduliitr Template is updated. Please update these lines.

**Review by sailesh-ramayanam — CHANGES_REQUESTED** [review]

File: solutions/functions/sum-of-multiples.js (hunk 1 above)
Comment (Abduliitr) [reply solutions/functions/sum-of-multiples.js:15]: Addressed

File: solutions/functions/sum-of-multiples.js (hunk 2 above)
Comment (Abduliitr) [reply solutions/functions/sum-of-multiples.js:30]: Addressed

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**